### DISCUSSION TITLE
The mythical griffin is a combination of which two creatures?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#445] [Galaxy Trivia] [answered]
**Bonus:** What popular strategy 🎮 game has a race with griffins?

**Comment by peterbaccaro** [comment]
The Griffin is a mythical creature that is a combination of an **Eagle** and a **Lion** (the Lion was traditionally considered the king of the beasts, and the Eagle the king of the birds).  The Griffin had the body, tail, and back legs of a lion; the head and wings of an eagle; and sometimes an eagle's talons as its front feet. 
**World of Warcraft** is the popular strategy game that has a race with griffins.

**Comment by ParanoidUser** [comment]
@BrianMasiga, would you like to try? 😊

**Reply by BrianMasiga** [reply]
The mythical griffin is a combination of a lion and an eagle. It has the head and wings of an eagle and the body of a lion.

**Comment by BrianMasiga** [comment, marked as answer]
> **Bonus:** What popular strategy 🎮 game has a race with griffins?

The popular strategy game that has a race with griffins is Warcraft III: Reign of Chaos.
